system/ufw.py

```diff
@@ -27,7 +27,7 @@
 short_description: Manage firewall with UFW
 description:
     - Manage firewall with UFW.
-version_added: 1.6
+version_added: 1.7
```

Comment:
`version_added` is not used for versioning the module, it is an information in which version of ansible the module was added. could you please change it back?